apps/3d_rec_framework/include/pcl/apps/3d_rec_framework/feature_wrapper/local/shot_local_estimator.h

```diff
@@ -73,7 +75,7 @@ namespace pcl
             mls.process(*filtered);
 
             processed.reset(new pcl::PointCloud<PointInT>);
-            normals.reset (new pcl::PointCloud<pcl::Normal>);
+            normals = pcl::make_shared<pcl::PointCloud<pcl::Normal>> ();
```

Comment:
The line above this one should also be replaced no?

Reply:
`PointInT` is a template parameter, I didn't test for that and apparently the check is failing with those. As is, I also have a failing test for when there is `typedef boost::shared_ptr<Derived> Ptr;`